tests/enforceToken/verbs.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/enforceToken/verbs.js
+ *
+ * Checks whether token-enforced path behaves as expected.
+*/
+
+const expect = require('chai').expect;
+const constants = require('../../api/v1/constants');
+const supertest = require('supertest');
+const api = supertest(require('../../index').app);
+const u = require('../testUtils');
+const registerPath = '/v1/register';
+const { OK, CREATED, FORBIDDEN } = constants.httpStatus;
+
+const STABLE_PATH = '/v1/subjects';
+const NEW_SUBJECT = 'NEW_SUBJECT';
+
+describe('API verb token enforced tests', () => {
+  it('GET docs without token should succeed', (done) => {
+    api.get('/v1/docs')
+    .expect(constants.httpStatus.OK)
```

Comment:
@iamigo @shriramshankar This is where the test fails: Error: expected 200 "OK", got 301 "Moved Permanently". Going to the page manually returns a 304 Not Modified, since I already loaded the page previously.
To work around it we can check that the error to be not defined. I am still looking into why this happens.